app/util/math.ts

```diff
@@ -58,9 +58,7 @@ export function round(num: number, digits: number) {
   return Number(nf.format(num))
 }
 
-// a separate function because I wanted to test it with a bunch of locales
-// to make sure the toLowerCase thing is ok
-export const toEngNotation = (num: number | bigint, locale = navigator.language) =>
+const toEngNotation = (num: number | bigint, locale?: string) =>
```

Comment:
Don't have to test this directly anymore because displayBigNum takes a locale arg too, so we don't have to export it at all.